data/topics.yml

```diff
@@ -104,6 +104,8 @@
       url: "paas/how-to-serve-s3-media-files"
     - title: "How do I update my SSL certificate on Aptible?"
       url: "paas/how-to-update-ssl-certificate"
+    - title: "How can I set up New Relic on Aptible?"
+      url: "paas/how-to-setup-new-relic"
```

Comment:
This needs to match the filename. (`how-to-setup-application-monitoring` != `how-to-setup-new-relic`)